src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const wifeOrHusband = sex === 'm'
+    ? (
+      <p className="Person__partner">
+        {partnerName}
+        {' '}
+        is my wife
+      </p>
+    )
+    : (
+      <p className="Person__partner">
+        {partnerName}
+        {' '}
+        is my husband
+      </p>
+    );
```

Comment:
1 - use interpolation
2 - no need to save paragraph to constant (especially no need to conditionally select needed paragraph if in both cases they are same), just add this paragraph to JSX layout. Better just to save needed string (wife or husband) using ternary operator